syntax = "proto3";

package NDTS.NTestingProcessor;

message TBrokerClientConfig {
    string server_url = 1;
    string queue_name = 2;
}
